src/main/java/bio/terra/service/resourcemanagement/google/GoogleProjectService.java

```diff
@@ -252,6 +255,15 @@ private GoogleProjectResource initializeProject(
         roleIdentityMapping, googleProjectId, PermissionOp.ENABLE_PERMISSIONS);
     resourceManagerService.addLabelsToProject(googleProjectResource.getGoogleProjectId(), labels);
 
+    String projectName;
+    switch (collectionType) {
+      case DATASET -> projectName = "TDR Dataset Project";
+      case SNAPSHOT -> projectName = "TDR Snapshot Project";
+      default -> throw new IllegalArgumentException("Invalid collection type");
+    }
```

Comment:
If you use a switch expression, you can eliminate `default`, since it will fail to compile if you omit a `case` for an enum member. Or, you can add these strings as a property on the enum, and avoid the `switch` completely.
```suggestion
    String projectName =
      switch (collectionType) {
        case DATASET -> "TDR Dataset Project";
        case SNAPSHOT -> "TDR Snapshot Project";
      }
```

Reply:
Oh interesting!  I'll go back and fix that